app.json

```diff
@@ -11,16 +11,8 @@
     "github"
   ],
   "env": {
-    "CONNECTED_APP_CALLBACK_URL": {
-      "description": "The callback url configured for the Connected App.  You should be able to get these values from the command `cumulusci2 org connected_app` in your locally configured instance of CumulusCI",
-      "value": ""
-    },
-    "CONNECTED_APP_CLIENT_ID": {
-      "description": "The Client ID of the Salesforce Connected App.  You should be able to get these values from the command `cumulusci2 org connected_app` in your locally configured instance of CumulusCI",
-      "value": ""
-    },
-    "CONNECTED_APP_CLIENT_SECRET": {
-      "description": "The Client Secret of the Salesforce Connected App. You should be able to get these values from the command `cumulusci2 org connected_app` in your locally configured instance of CumulusCI",
+    "DB_ENCRYPTION_KEYS": {
```

Comment:
Why plural?

Reply:
After reviewing metaci/tests/test_fields.py I think I understand why but want to use this to open up a brief discussion on this.

Reply:
Does this sentence in the pull request description help clarify? "DB_ENCRYPTION_KEYS can be a newline-separated list of keys; the keys can be rotated by adding a new one at the beginning, but old keys need to remain in order to decrypt data that was stored using them."